apps/a2/a2/__init__.py

```
@@ -0,0 +1,3 @@
+from l2 import hello
+def main() -> None:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 0

(E302)